go.mod

```diff
@@ -3,6 +3,9 @@ module github.com/IBM/ibm-vpc-block-csi-driver-operator
 go 1.16
 
 require (
+	bou.ke/monkey v1.0.2
```

Comment:
Please re-run `go mod vendor` and don't forget to add newly introduced files, this pkg is missing there.